models/bootcampers.js

```diff
@@ -0,0 +1,41 @@
+import { pool } from '../db/index.js'
+
+export async function getAllBootcampers () {
```

Comment:
Very minor point, and not really important for your application, but when having endpoints for fetching all of something it is often useful to have a query parameter which allows you to paginate the response. E.g if I only want to retrieve the first 100 bootcampers, I wouldn't have to retrieve all 1 million of them and filter that on the front end.